### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by Sameeratweb**
### Requesting to submit a pull request to the PyVerse repository.

---

#### Issue Title
**Please enter the title of the issue related to your pull request.**  
*Object size detection using python*

- [X] I have provided the issue title.

---

#### Info about the Related Issue
**What's the goal of the project?**  
*This code creates an app where users can upload an image to measure the size of objects in it. After uploading, the app detects objects, measures their height and width based on a reference size, and displays the image with dimensions marked on each object. It also shows the total number of objects found and lists each object’s size. *

- [x] I have described the aim of the project.

---

#### Name
**Please mention your name.**  
*Sameer Desai*

- [X] I have provided my name.

---

#### GitHub ID
**Please mention your GitHub ID.**  
*Sameeratweb*

- [X] I have provided my GitHub ID.

---

#### Email ID
**Please mention your email ID for further communication.**  
*sameerdesai.dev@gmail.com*

- [X] I have provided my email ID.

---

#### Identify Yourself
**Mention in which program you are contributing (e.g., WoB, GSSOC, SSOC, SWOC).**  
*GSSOC*

- [X] I have mentioned my participant role.

---

#### Closes
**Enter the issue number that will be closed through this PR.**  
*Closes: #968 *

- [X] I have provided the issue number.

---

#### Describe the Add-ons or Changes You've Made
**Give a clear description of what you have added or modified.**  
*added a code that detects size of an object*

- [X] I have described my changes.

---

#### Type of Change
**Select the type of change:**  
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

---

#### How Has This Been Tested?
**Describe how your changes have been tested.**  
*tested on my local machine by running the server*

- [X] I have described my testing process.

---

#### Checklist
**Please confirm the following:**  
- [X] My code follows the guidelines of this project.
- [X] I have performed a self-review of my own code.
- [X] I have commented my code, particularly wherever it was hard to understand.
- [X] I have made corresponding changes to the documentation.
- [X] My changes generate no new warnings.
- [X] I have added things that prove my fix is effective or that my feature works.
- [X] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

File: Computer Vision/Object Size Detection/README.md
Comment (UTSAVS26) [inline Computer Vision/Object Size Detection/README.md:1]: Add a detailed README file.

**Comment by Sameeratweb** [comment]
I have made the necessary changes.

**Comment by Sameeratweb** [comment]
please review the latest changes

**Review by TheChaoticor — APPROVED** [review]
Good clean code

**Pull request merged**